packages/fluentui/react/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -73,18 +73,20 @@ const dropdownStyles: ComponentSlotStylesPrepared<DropdownPropsAndState, Dropdow
     borderRadius: v.containerBorderRadius,
     ...(p.open && p.position === 'above' && { borderRadius: v.openAboveContainerBorderRadius }),
     ...(p.open && p.position === 'below' && { borderRadius: v.openBelowContainerBorderRadius }),
-    ':hover': {
-      backgroundColor: v.backgroundColorHover,
-      borderColor: v.borderColorHover,
+    ...(!p.disabled && {
```

Comment:
Is there any way to avoid negative conditions? I.e.

```tsx
':hover': {
   ...p.disabled && {
      background: v.disabledBackgroundColor,
      borderColor: v.disabledBorderColor,
   }
}
```

Reply:
well, it's `!p.disabled` so I don't know how to negate that. any suggestions?

Reply:
I expanded it below a bit:

```tsx
':hover': {
      backgroundColor: v.backgroundColorHover,
      borderColor: v.borderColorHover,

      ...(p.open && {
        borderColor: v.openBorderColorHover
      }),

      ...p.disabled  && {
         backgroundColor: v.backgroundColor,
         borderColor: v.borderColor, // this will color to the default one, may be a separate variable `disabledBorderColorHover`
      },

      [`& .${Dropdown.slotClassNames.triggerButton}`]: {
        color: v.triggerButtonColorHover,

        ...p.disabled && {
          color: v.color, // may be a separate variable `disabledTriggerColorHover`          
        }
      }
    },
```


Why it's better that `!disabled`?
- it's closer to CSS
- it creates valid variables for the state, if we will need different values for HC theme it will be easy to add them